File: android_apps/tensorflow_for_poets_tflite/app/src/main/java/com/scanfit/modeling/Person.java
package com.scanfit.modeling;

import java.io.Serializable;
import java.util.Date;

public class Person implements Serializable {
    public String gender;
    public BodyType bodyType;
    public Date birthday;
    public Weight weight;
    public Height height;

    public Person(String gender, BodyType bodyType, Date birthday) {

    }
}

File: android_apps/tensorflow_for_poets_tflite/app/src/main/java/com/scanfit/modeling/Person.kt
package com.scanfit.modeling

import java.io.Serializable
import java.util.*

class Person(name: String, username: String, email: String, gender: String, bodyType: BodyType, birthday: Date) : Serializable, UserInfo {


    override var username: String = username
    override var name: String = name
    override var email: String = email

    var gender: String? = null
    var bodyType: BodyType? = null
    var birthday: Date? = null
    var weight: Weight? = null
    var height: Height? = null
}
